frontend/scripts/react-components/dashboard-element/dashboard-widget/dashboard-widget.component.jsx

```diff
@@ -5,44 +5,76 @@ import Spinner from 'react-components/shared/shrinking-spinner/shrinking-spinner
 import DashboardWidgetLabel from 'react-components/dashboard-element/dashboard-widget/dashboard-widget-label.component';
 import DashboardWidgetLegend from 'react-components/dashboard-element/dashboard-widget/dashboard-widget-legend/dashboard-widget-legend.component';
 import ErrorCatch from 'react-components/shared/error-catch.component';
-
+import Text from 'react-components/shared/text';
+import Heading from 'react-components/shared/heading';
+import DynamicSentenceWidget from 'react-components/dynamic-sentence-widget';
 import 'react-components/dashboard-element/dashboard-widget/dashboard-widget.scss';
 
+const renderChart = ({ data, chartConfig, dynamicSentenceParts }) => {
+  if (chartConfig.type === 'sentence') {
+    return (
+      <div className="widget-centered">
+        <DynamicSentenceWidget
+          data={data}
+          config={chartConfig}
+          dynamicSentenceParts={dynamicSentenceParts}
+        />
+      </div>
+    );
+  }
+  return (
+    <React.Fragment>
+      <DashboardWidgetLegend colors={chartConfig.colors} />
+      {chartConfig.yAxisLabel && (
+        <DashboardWidgetLabel
+          text={chartConfig.yAxisLabel.text}
+          suffix={chartConfig.yAxisLabel.suffix}
+        />
+      )}
+      <Chart className="widget-chart" data={data} config={chartConfig} />
+    </React.Fragment>
+  );
+};
+
+renderChart.propTypes = {
```

Comment:
I know that the linter complains and thinks its a component, I don't have an awesome solution for this, I generally add an eslint-disable comment. 😞